strings/ctype-czech.c

```diff
@@ -42,8 +42,8 @@
 	passes, that's why we need four times more space for expanded
 	string.
 
-	This file also contains the ISO-Latin-2 definitions of
```

Comment:
This comment refers to what the file implements. Please don't change it.

Reply:
The existing comment was rather ambiguous: what exactly does "contains" means?

> ```
> This file also contains the ISO-Latin-2 definitions of
> ```

The new version of the comment is much clearer and less ambiguous.